test/llvm-intrinsics/add_sub.sat.ll

```diff
@@ -1,6 +1,16 @@
 ; RUN: llvm-as < %s -o %t.bc
 ; RUN: llvm-spirv %t.bc -o %t.spv
-; RUN: llvm-spirv %t.spv -to-text -o - | FileCheck %s
+; RUN: llvm-spirv %t.spv -to-text -o - | FileCheck %s --check-prefixes=COMMON,OPENCL
+; RUN: spirv-val %t.spv
+
+; RUN: llvm-as < %s -o %t.bc
```

Comment:
excessive - we've already assembled the *.ll file, no need to assemble again (here and on line 11)

Reply:
oh, indeed, thanks!